### PR TITLE
Support Ascii Union and Intersection

### LINKED ISSUE #10414: `pprint(Union, use_unicode=False)` raises error (but `str(Union)` works)
See `printing/pretty/pretty.py`.

1.  Possible fix:
`union_delimiter = ' %s ' % pretty_atom('Union', 'U')`

2.  Similarly, I wonder if lower-case `n` might be an appropriate ascii character for `Intersection`?  If not, at least throw `NotImplemented` like in `SymmetricDifference`.

### PR DESCRIPTION
**Pull request by tokencolour**
Fixes #10414

### CODE REVIEW

File: sympy/printing/pretty/tests/test_pretty.py [hunk 1]
```diff
@@ -3139,6 +3139,22 @@ def test_pretty_ComplexRegion():
     ucode_str = u('{r⋅(ⅈ⋅sin(θ) + cos(θ)) | r, θ ∊ [0, 1] × [0, 2⋅π)}')
     assert upretty(ComplexRegion(Interval(0, 1)*Interval(0, 2*pi), polar=True)) == ucode_str
 
+def test_pretty_Union():
+    from sympy import Union, Interval
```
Comment (cbm755) [inline sympy/printing/pretty/tests/test_pretty.py]: Maybe done at top of file?  If so, remove

File: sympy/printing/pretty/tests/test_pretty.py [hunk 2]
```diff
@@ -3139,6 +3139,22 @@ def test_pretty_ComplexRegion():
...
+    a, b = Interval(2, 3), Interval(4, 7)
+    ucode_str = u('[2, 3] ∪ [4, 7]')
+    ascii_str = '[2, 3] U [4, 7]'
+    assert upretty(Union(a, b)) == ucode_str
+    assert pretty(Union(a, b)) == ascii_str
+
+def test_pretty_Intersection():
+    from sympy import Intersection, Interval, symbols
```
Comment (cbm755) [inline sympy/printing/pretty/tests/test_pretty.py]: same here

**Comment by cbm755** [comment]
Minor comments above.  Personally, I like the test to point back to the issue number, either in the test name or in a comment, but I forget if the development guide offers advice on this...

I restarted the two failing tests (which look unrelated to me).

+1

**Comment by tokencolour** [comment]
@cbm755 , Thank you for reviewing this. I have made the changes that you suggested. Kindly take a look at this.

**Comment by cbm755** [comment]
+1.  I will merge in 24 hours or so.

@gxyd @hargup Please say so if you don't want `n` as the Intersection character for ASCII pprint.

**Comment by hargup** [comment]
I'm +1. @tokencolour thanks for taking this up.

**Comment by tokencolour** [comment]
Thank you!

**Pull request merged**